test/molecule-role/molecule/integrations/molecule.yml

```diff
@@ -23,6 +23,9 @@ provisioner:
     destroy: destroy.yml
   lint:
     name: ansible-lint
+  inventory:
```

Comment:
i see wuite some changes to molecule, but as far as i can see those run still on py3, why was this needed, or were these pulled in form some branch?

Reply:
this allows us to use group_vars instead of keeping including common_vars.yaml
I'm rolling out this change to all scenarios